### DISCUSSION TITLE
Make configuration properties price relevant?

### DISCUSSION THREAD

**Discussion by javac9** [opencrx/opencrx-documentation#17] [Q&A]
Are there plans to make configuration properties price relevant in the future? On the following screenshot it was explained that they aren't price relevant. I got an impression that OpenCrx thrives to be as flexible as possible, I believe it would make sense to (try to) implement this in the future.

Another reasoning is that many of us have for sure, when buying on eg. Amazon, experienced situations where a certain product costs more if it's of some special color, or shape or whatever else which actually doesn't qualify it to be maintained as a separate product.

![https://i.ibb.co/KwGjpVw/product1.png](https://i.ibb.co/KwGjpVw/product1.png)

**Comment by wfro64** [comment]
This is a documentation issue. Custom-specific PricingRules and DiscountSchedules allow to base pricing on any data, including ProductConfiguration, etc.

- http://www.opencrx.org/opencrx/4.3/uml/opencrx-core/product1/tn/999_PricingRule.png.html
- http://www.opencrx.org/opencrx/4.3/uml/opencrx-core/product1/tn/999_DiscountSchedule.png.html

However, the default pricing rule is limited in that regards.
